### PR TITLE
DCS-1303 add location banner

### PR DESCRIPTION
**Pull request by louisenorris**

### CODE REVIEW

File: server/middleware/setUpCurrentUser.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ export default function setUpCurrentUser(services: Services): Router {
   const router = Router({ mergeParams: true })
 
   router.use(auth.authenticationMiddleware(tokenVerifier))
-  router.use(populateCurrentUser(services.userService))
+  router.use(populateCurrentUser(services.userService, services.expectedArrivalsService))
```
Comment (andrewrlee) [inline server/middleware/setUpCurrentUser.ts:14]: umm.. this seems a bit off - populateCurrentUser is domain agnostic and we're pushing in the thing responsible for a chunk of our domain.

The only reason we need this is for: 
`expectedArrivalsServices.getPrison(user.activeCaseLoadId)`

the concept of `getPrison` doesn't seem very related to conceptual area of expected arrivals.

Maybe we should extract it into its own service

Comment (louisenorris) [reply server/middleware/setUpCurrentUser.ts]: Sorted

**Review by andrewrlee — DISMISSED** [review]
LGTM

**Review by andrewrlee — DISMISSED** [review]
LGTM

File: server/views/partials/header.njk [hunk 2]
```diff
@@ -245,3 +246,18 @@
...
+        {% if phaseName %}
+            {{ govukTag({
+                text: phaseName,
+                classes: 'govuk-!-margin-right-1 ' + phaseNameColour
+            }) }}
+        {% endif %}
+        <span class="gtm-active-case-load" data-qa="active-case-load">{{ user.activeCaseLoad.description }}</span>
+    </span>
+    {% if user.userCaseLoads.length > 1 %}
+      <a class="location-bar__link" href="https://digital-dev.prison.service.justice.gov.uk/change-caseload" data-qa="change-location-link">Change your location</a>
```
Comment (andrewrlee) [inline server/views/partials/header.njk:261]: this needs to be parameterised

File: server/config.ts [hunk 3]
```diff
@@ -35,6 +35,7 @@ export default {
   https: production,
   staticResourceCacheDuration: 20,
   phaseName: process.env.SYSTEM_PHASE,
+  dpsUrl: get('DPS_URL', 'http://localhost:3000', requiredInProduction),
```
Comment (louisenorris) [inline server/config.ts:38]: Not sure is localhost on 300 is correct here

File: server/views/partials/header.njk (hunk 2 above)
Comment (louisenorris) [reply server/views/partials/header.njk]: Nope, I've added one though

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**